scripts/SREP/example/metadata.yaml

```diff
@@ -1,7 +1,7 @@
 file: script.sh
-name: example # name should be the same as the subdir, eg: SRE/example in this case
-description: Example script
-author: qhua948
+name: get-targets-down # name should be the same as the subdir, eg: SRE/example in this case
+description: List pods marked as down by cluster Prometheus
+author: mdewald
```

Comment:
Why you are changing this file?

Reply:
he changed the wrong file..
https://github.com/openshift/managed-scripts/pull/34/files#diff-040954bd9ddd54f7a4cec2401a7fbae97bc6b5bc76b0c19d518c8d7507e27f57R2

Reply:
:facepalm: